test/message-generator/test/interop-proxy-with-multi-ups-extended-jn.json

```diff
@@ -68,6 +68,11 @@
                 }
             }
         },
+        {
+            "command": "sleep",
+            "args": ["1000"],
```

Comment:
Why do we need to sleep for 1000s?

Reply:
Short answer: I'm not sure.
I believe this was included on main when I started work, was since deleted, and when I rebased, it was just added in from what I had locally.  I can delete them.